store/localcachelayer/channel_layer.go

```diff
@@ -70,6 +80,30 @@ func (s LocalCacheChannelStore) InvalidateGuestCount(channelId string) {
 	}
 }
 
+func (s LocalCacheChannelStore) InvalidateAllChannelMembersNotifyPropsForChannel(channelId string) {
```

Comment:
You are changing the method name here, so you are not overriding the behavior. And, this function is already doing sending the cache cancelation message to the cluster, so you have to modify the `app/web_hub.go` to not send the cluster invalidation, because is no longer needed there (for this cache).

Reply:
Ok, I will fix it. Thank you.